ca-functional/src/test/resources/features/privateEndpoint.feature

```diff
@@ -0,0 +1,6 @@
+Feature: testing that private endpoint can be reached
+  Scenario: client calls GET /private/status
```

Comment:
You could consider adding a bit more information to the scenario - For example, is it a happy case or a sad case? It helps makes it explicit when looking for which particular test has failed

Reply:
Noted @zyrondias - makes sense. @rosa03 and I will look to add.